src/doc/rustc/src/platform-support.md

```diff
@@ -179,6 +179,7 @@ target | std | notes
 `wasm32-unknown-emscripten` | ✓ | WebAssembly via Emscripten
 `wasm32-unknown-unknown` | ✓ | WebAssembly
 `wasm32-wasi` | ✓ | WebAssembly with WASI
+`wasm32-wasi-preview1-threads` | ✓ | WebAssembly with WASI Preview 1 and threads
```

Comment:
Can you make this a link to a separate target doc like many of the other, newer targets do? (See a few lines below for what I mean)

It would be good to document the target maintainers there and it could also mention some the experimental nature of the target.

Reply:
✅